x-pack/plugins/apm/public/components/app/service_dashboards/index.tsx

```diff
@@ -49,7 +49,10 @@ export function ServiceDashboards() {
   const {
     path: { serviceName },
     query: { environment, kuery, rangeFrom, rangeTo, dashboardId },
-  } = useApmParams('/services/{serviceName}/dashboards');
+  } = useAnyOfApmParams(
```

Comment:
@cauemarcondes the above comment related to the `useAnyOfApmParams` also apply to this component?  

Asking cause this requires a bigger refactoring on how we reuse components between services and mobile services. (ex alerts tab)

Reply:
If that needs too much work, I'd probably do it only for `x-pack/plugins/apm/public/components/app/service_dashboards/actions/save_dashboard_modal.tsx` in this case, it's already a good start.

Reply:
Unfortunately, it does 😒  I'll update just the `save_dashboard_modal` for now